pkg/machinepool/validations/disk_size.go

```diff
@@ -17,6 +17,9 @@ const (
 	// machinePoolRootVolumeSizeMaxAsOf414 is the maximum size of the root volume as of 4.14
 	// 16 TiB - limit as of 4.14
 	machinePoolRootVolumeSizeMaxAsOf414 = 16384
```

Comment:
Isn't that the same value?

Reply:
Oh, is there a difference from topology or is it only an OCP thing? Should we separate just to be safer?

Reply:
we just want to keep them separate in case we want to change the requirements to be different for HCP. the minimum specifically may be lower than classic but for now I'm just using the same values as classic so we can get the API changes in sooner